ui/hooks/use-keys.ts

```diff
@@ -1,16 +1,37 @@
 import { useEffect, useState } from "react";
 
+export function hasKeys(expectedKeys: string[], pressedKeys: Map<String, boolean> ) {
+  return expectedKeys.every(k => pressedKeys.has(k));
+};
+
 export default function useKeys() {
   const [keys, setKeys] = useState(new Map());
+  const hotKeys = ["Control", "Enter", "Escape", "s", "t", "w"];
+
+  function copyKeyMap(map:Map<KeyboardEvent, boolean>) {
```

Comment:
Instead of this function, we could do `const newMap = new Map(oldMap)`.

Reply:
Good idea. I updated the PR